addons/osm-edge/disable

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
When the addon is disabled the `pod/osm-controller` is not removed so its status is always set to enabled.

Reply:
this shouldn't be the case, as `disable` script invoks `osm uninstall mesh` which will remove controller and other related pods. I couldn't reproduce this behavior on local boxes, can you please advise the steps which you followed to see this behavior?

Reply:
Here is what I get when I try to use osm:
```
ubuntu@ip-172-31-13-141:~$ /var/snap/microk8s/current/bin/osm uninstall mesh

List of meshes present in the cluster:
Unable to list meshes within the cluster.
Error: Could not list deployments Get "http://localhost:8080/apis/apps/v1/deployments?labelSelector=app%3Dosm-controller": dial tcp 127.0.0.1:8080: connect: connection refused
```

Is it possible the osm is trying to find the kubeconfig file in a defualt location?

Reply:
It seems so:
```
ubuntu@ip-172-31-13-141:~$ mkdir ~/.kube
ubuntu@ip-172-31-13-141:~$ sudo microk8s.config > ~/.kube/config
ubuntu@ip-172-31-13-141:~$ /var/snap/microk8s/current/bin/osm uninstall mesh

List of meshes present in the cluster:

MESH NAME   MESH NAMESPACE   VERSION   ADDED NAMESPACES
osm         osm-system       v1.1.0    

MESH NAME   MESH NAMESPACE   SMI SUPPORTED
osm         osm-system       HTTPRouteGroup:v1alpha4,TCPRoute:v1alpha4,TrafficSplit:v1alpha2,TrafficTarget:v1alpha3

To list the OSM controller pods for a mesh, please run the following command passing in the mesh's namespace
        kubectl get pods -n <osm-mesh-namespace> -l app=osm-controller

Uninstall OSM [mesh name: osm] in namespace [osm-system] and/or OSM resources ? [y/n]: ^C
```

Reply:
Here is something that worked:
```
sudo KUBECONFIG=/var/snap/microk8s/current/credentials/client.config  /var/snap/microk8s/current/bin/osm uninstall mesh
```

Reply:
Yeah, just figured that `disable` script was missing the location of `KUBECONFIG`. I've updated `disable` script and added `KUBECONFIG`.